plugins/mcreator-localization/lang/texts.properties

```diff
@@ -893,6 +893,7 @@ blockly.block.entity_get_spawn_x=Get X spawn coordinate of %1 for its current wo
 blockly.block.entity_get_spawn_y=Get Y spawn coordinate of %1 for its current world %2
 blockly.block.entity_get_spawn_z=Get Z spawn coordinate of %1 for its current world %2
 blockly.block.entity_get_stepheight=Get step height of %1
+blockly.block.entity_get_string_uuid=Get %1 UUID as string
```

Comment:
String is already "implicitely" shown by block color, so something like `Get UUID of %1` would work better and also make block thinner as full inline block with input after all text

Reply:
Ok